Content/Items/BaseTypes/SmartAccessory.cs

```diff
@@ -37,6 +37,20 @@ protected SmartAccessory(string name, string tooltip) : base()
 			this.tooltip = tooltip;
 		}
 
+		public override ModItem Clone(Item newEntity)
+		{
+			var clone = base.Clone(newEntity) as SmartAccessory;
+
+			var newList = new List<Item>();
+			foreach(Item item in parents)
+			{
+				newList.Add(item.Clone());
```

Comment:
This seems at odds with the implementation of `Simulate`. Items in the `parents` list are meant to also be in `mp.simulatedAccessories`, and there is only meant to be one per type.

I would assume then that it would be okay here to share `Item` instances in the `parents` list. You could use `clone.parents = new List<Item>(parents)`